src/lib/OS/SyscallHandler.cc

```diff
@@ -16,8 +18,9 @@ SyscallHandler::SyscallHandler(SimOS* OS,
   // Define vector of all currently supported special file paths & files.
   supportedSpecialFiles_.insert(
       supportedSpecialFiles_.end(),
-      {"/proc/cpuinfo", "proc/stat", "/sys/devices/system/cpu",
-       "/sys/devices/system/cpu/online", "core_id", "physical_package_id"});
+      {"/proc/cpuinfo", "proc/stat", "proc/self/maps", "maps",
```

Comment:
what's "maps"?

Reply:
`maps` refers to the same file as `proc/self/maps`. But, within `proc/self/maps` self is translated to the calling processes TGID (PID). As such, we need to ensure we support both `proc/self/maps` and paths such as `proc/1/maps`. 

A similar approach to `core_id` has been taken where rather than support all numbers, we just indicate the file name itself as also being valid.